values-ar/strings.xml

```diff
@@ -0,0 +1,142 @@
+            <string name="label_variant">النوع</string>
+            <string name="invalid_combo">دمج غير صالح</string>
+            <string name="label_install">تثبيت</string>
+            <string name="label_update">تحديث</string>
+            <string name="label_change_selection">تغيير الاختيار</string>
+            <string name="download_started">بدء التنزيل</string>
+            <string name="label_download_package">تنزيل الحزمة</string>
+            <string name="label_install_package">تثبيت الحزمة</string>
+            <string name="label_md5_checksum">MD5-Checksum</string>
+            <string name="label_copy_checksum">نسخ تدقيق المجموع</string>
+            <string name="file_not_found">الملف غير موجود</string>
+            <string name="label_versionlog">سجل النسخة</string>
+            <string name="label_show_md5">عرض MD5-Checksum</string>
+            <string name="label_show_versionlog">عرض سجل النسخة</string>
+            <string name="label_share">مشاركة</string>
+            <string name="label_new">جديد</string>
+            <string name="label_checksum_valid">تدقيق المجموع صحيح</string>
+            <string name="label_checksum_invalid">تدقيق المجموع غير صحيح</string>
+            <string name="label_starting_download">بدء التنزيل</string>
+        <!--EXPLANATION-->
+            <string name="explanation_optional_app_update">يوجد تحديث للتطبيق متوفر</string>
+            <string name="explanation_forced_update">يجب عليك التحديث من أجل الاستمرار في استخدام التطبيق</string>
+            <string name="label_root_is_need_for_rootmode">يحتاج الجهاز إلى الى الروت من أجل التفعيل</string>
+            <string name="autoinstall_root_disclaimer">التثبيت التلقائي يعمل فقط إذا تم تثبيت Open GApps كتطبيق نظام أو بإذن الروت</string>
+
+    <!--______SUPPORT______-->
+        <string name="label_app">التطبيق</string>
+        <string name="label_homepage">الصفحة الرئيسية</string>
+        <string name="label_translate">الترجمة</string>
+        <string name="label_support">الدعم</string>
+        <string name="label_wiki">Open GApps Wiki</string>
+        <string name="label_support_forum">منديات الدعم</string>
+        <string name="label_support_chat">دردشات الدعم</string>
+        <string name="label_sourcecode">الشفرة المصدرية</string>
+        <string name="label_support_opengapps">دعم مشروع Open GApps</string>
+        <string name="label_support_opengapps_description">مشروع المصدر المفتوح يوفر النصوص لحزم OpenGApps.org</string>
+        <string name="label_report_bug">الإبلاغ عن خطأ</string>
+
+    <!--______ABOUT______-->
+        <!--LABELS-->
+            <string name="title_activity_about">حول</string>
+            <string name="label_version">الإصدار</string>
+            <string name="main_dev">المطور الرئيسي</string>
+            <string name="label_design">العمل الفني</string>
+            <string name="label_producer">منتج</string>
+            <string name="label_easteregg_found">تعريب احمد الزهراني @silent_6600</string>
```

Comment:
Thanks! :-)

I filed issue https://github.com/opengapps/opengapps-app-support/issues/19 for showing the names in a future version :-)